app/policies/application_policy.rb

```diff
@@ -0,0 +1,45 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    scope.where(id: record.id).exists?
+  end
```

Comment:
Ok, please do not change the default `ApplicationPolicy` from `false` to anything else. Especially that Authorization should not care about whenever record exists or not - that's why `Model.find()` raises `NotFound` error when record with given id does not.